### PR TITLE
Fix lint errors & get virtcontainers tests running

### PR DESCRIPTION
**Pull request by jodh-intel**
Fix all lint errors reported by the `.ci/static-checks.sh` script in the `tests` repo and get the `virtcontainers` tests working.

---

This is still a work-in-progress. Known issues:

- test timeout: it appears that virtcontainers needs a 60s timeout, rather than the default 10s in https://github.com/kata-containers/tests/blob/master/.ci/go-test.sh#L14. As shown, that could be handled via an env var.
- I'm seeing a hook tests failing locally but have run out of time today to debug:
  ```
  --- FAIL: TestRunHook (0.00s)
        hook_test.go:84: unexpected failure: exit status 1: stdout: , stderr:
  === RUN   TestRunHookTimeout
  --- FAIL: TestRunHookTimeout (0.00s)
        hook_test.go:84: unexpected failure: exit status 1: stdout: , stderr:
  ```

### CODE REVIEW

File: cli/create_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 
 	vc "github.com/kata-containers/runtime/virtcontainers"
 	"github.com/kata-containers/runtime/virtcontainers/pkg/oci"
-	"github.com/kata-containers/runtime/virtcontainers/pkg/vcMock"
+	"github.com/kata-containers/runtime/virtcontainers/pkg/vcmock"
```
Comment (sboeuf) [inline cli/create_test.go]: Why did `golint` recommended this ? I thought Golang was all about camel case ?

Comment (jodh-intel) [reply cli/create_test.go]: Here's the message:

```
virtcontainers/pkg/vcMock/container.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/mock.go:25:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/mock_test.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/pod.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/types.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
virtcontainers/pkg/vcMock/utils.go:15:1:warning: don't use MixedCaps in package name; vcMock should be vcmock (golint)
```

**Comment by jodh-intel** [comment]
Tests all passed so branch ready to review.

File: cli/create_test.go (hunk 1 above)
Comment (sboeuf) [reply cli/create_test.go]: Okay, fair enough

**Comment by sboeuf** [comment]
LGTM

File: virtcontainers/qemu.go [hunk 2]
```diff
@@ -559,19 +559,6 @@ func (q *qemu) addDeviceToBridge(ID string) (string, string, error) {
 	return "", "", err
 }
 
-func (q *qemu) removeDeviceFromBridge(ID string) error {
```
Comment (devimc) [inline virtcontainers/qemu.go]: ok, that's weird, if AddDeviceToBridge is called [here](https://github.com/jodh-intel/runtimes/blob/734c387f4ed7bbcb993c5029ec5266468b1cdb4f/virtcontainers/qemu.go#L585) , this function MUST BE called when the device is removed , seems like something is broken

**Comment by devimc** [comment]
LGTM

File: virtcontainers/qemu.go (hunk 2 above)
Comment (jodh-intel) [reply virtcontainers/qemu.go]: /cc @amshinde, @sameo.

Comment (sboeuf) [reply virtcontainers/qemu.go]: @devimc that's a good point, we should be unplugging the devices from the bridge.

**Comment by jodh-intel** [comment]
Yes, we need some feedback from @amshinde on the `removeDeviceFromBridge()` deletion above.

**Comment by amshinde** [comment]
@jodh-intel You are right, we need to call removeDeviceBridge for virtio-blk case. Will look into this. I'll raise a another PR to address this case.

**Comment by jodh-intel** [comment]
Hi @amshinde - are you happy for this PR to be merged and for you to re-add `removeDeviceBridge()` in another PR?

**Comment by amshinde** [comment]
@jodh-intel Yeah, that works for me.

**Comment by amshinde** [comment]
I'll go ahead and merge this one.

**Comment by amshinde** [comment]
@sboeuf Beat me to it :)

**Comment by sboeuf** [comment]
@amshinde I was faster haha ! Could you please an issue to make sure we don't forget to fix this.

**Pull request merged**